docs/en/siem/cases-api-find-connectors.asciidoc

```diff
@@ -0,0 +1,96 @@
+[[cases-api-find-connectors]]
```

Comment:
because of that https://github.com/elastic/kibana/blob/master/x-pack/plugins/case/server/routes/api/cases/configure/get_connectors.ts#L33, we will only return connectors who are the type '.servicenow'